dir.props

```diff
@@ -72,6 +72,7 @@
        -->
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/packages/index.json;
```

Comment:
Is OverridePackageSource intentionally at the bottem of this list?

Reply:
I think even in the context of a full build, we would want it to be at the top. @weshaggard ?

Reply:
We should probably put the OverridePackageSource at the top.